cmd/rad/cmd/root.go

```diff
@@ -53,9 +53,15 @@ import (
 	recipe_register "github.com/radius-project/radius/pkg/cli/cmd/recipe/register"
 	recipe_show "github.com/radius-project/radius/pkg/cli/cmd/recipe/show"
 	recipe_unregister "github.com/radius-project/radius/pkg/cli/cmd/recipe/unregister"
+	resource_create "github.com/radius-project/radius/pkg/cli/cmd/resource/create"
 	resource_delete "github.com/radius-project/radius/pkg/cli/cmd/resource/delete"
 	resource_list "github.com/radius-project/radius/pkg/cli/cmd/resource/list"
 	resource_show "github.com/radius-project/radius/pkg/cli/cmd/resource/show"
+	resourceprovider_delete "github.com/radius-project/radius/pkg/cli/cmd/resourceprovider/delete"
+	resourceprovider_list "github.com/radius-project/radius/pkg/cli/cmd/resourceprovider/list"
+	resourceprovider_show "github.com/radius-project/radius/pkg/cli/cmd/resourceprovider/show"
+	resourcetype_list "github.com/radius-project/radius/pkg/cli/cmd/resourcetype/list"
+	resourcetype_show "github.com/radius-project/radius/pkg/cli/cmd/resourcetype/show"
```

Comment:
Have we added **createOrUpdate** for **resourceProvider** and **resourceType**?

Reply:
This will be in a separate PR. Please read the PR description.